frontend/src/pages/setting/powers.tsx

```diff
@@ -0,0 +1,37 @@
+import VerticalSidebar from "../../components/VerticalSidebar";
+import SlidebarSettingPage from "../../components/SlidebarSettingPage";
+
+
+function PowersPage() {
+
+    return (<>
+
+        <VerticalSidebar />
```

Comment:
maybe consider make this into a wrapper that contain VerticalSidebar instead?

imagine having to adding more component but having to repeat multiple time